public/docs/ts/latest/guide/architecture.jade

```diff
@@ -160,23 +160,26 @@ figure
 
 :marked
   A _component_ controls a patch of screen called a *view*.
-  The shell at the application root with navigation links, a list of heroes, and a hero editor are all views controlled by components.
+
+  For example, the following views are controlled by components:
+
+  * The app root with the navigation links.
```

Comment:
This made me look up the writing guidelines, and sure enough, each bulleted list item is supposed to end with a period. (That's something I haven't been doing.) Good going!